BootloaderCorePkg/Tools/GenExtKeyHashStore.py

```diff
@@ -0,0 +1,91 @@
+## @ GenExtKeyHashStore.py
+# Tool to generate an public key hash store.
+# Generated binary can be updated using capsule firmware update
+#
+#
+# Copyright (c) 2020, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+import os
+import argparse
+import subprocess
+import sys
+import struct
+import binascii
+import string
+from ctypes import *

```

Comment:
Please remove unnecessary imports, binascii, ctypes, etc

Reply:
Done. Removed imports which are not required.